crates/rpc/utils.rs

```diff
@@ -34,6 +35,11 @@ impl From<RpcErr> for RpcErrorMetadata {
                 data: None,
                 message: "Unsupported fork".to_string(),
             },
+            RpcErr::BadHexFormat => RpcErrorMetadata {
+                code: -32602,
+                data: None,
+                message: "invalid argument 0: hex string without 0x prefix".to_string(),
```

Comment:
I guess the error won't always be in argument 0. Maybe we can add a field to `BadHexFormat` to store the index of the argument?

Reply:
Yes, that makes sense. I'll make the change